src/renderer/components/deposit/add/AsymDeposit.tsx

```diff
@@ -104,14 +112,34 @@ export const AsymDeposit: React.FC<Props> = (props) => {
     [oAssetBalance]
   )
 
+  const chainFees$ = useMemo(() => fees$, [fees$])
+
+  const depositFeesParams: AsymDepositFeesParams = useMemo(
+    () => ({
+      asset,
+      amount: assetAmountToDeposit,
+      memo: oMemo,
+      recipient: oPoolAddress,
+      type: 'asym'
+    }),
+    [asset, assetAmountToDeposit, oMemo, oPoolAddress]
+  )
+
+  const [depositFeesRD] = useObservableState<DepositFeesRD>(() => chainFees$(depositFeesParams), RD.initial)
+
+  // reload fees whenever params have been changed
+  useEffect(() => {
+    reloadFees(depositFeesParams)
+  }, [depositFeesParams, reloadFees])
+
```

Comment:
Pls check latest changes of `swap$` - we reload fees **after** changing assets or input values (amount/memo) now. With it we avoid some uneeded requests to get fees. _It can be done with another PR..._

Reply:
yep. here is the same idea but in another place. at the `fees/deposit.ts` (your next comment) with `distinctUntilChanged`. just because deposit fees  are used in 2 separate places (sym+asym) it was moved to the service-layer to have the same logic for different places

Reply:
created #1042

Reply:
`distinctUntilChanged  is good, but not enough. Imagine an user change his amount in the input field. With any small change, a request to load fees will be done. We can minimize such requests by adding a `onBlur` handler to the input to re-load fees only, if the user "leaves" the input.

Reply:
Resolved by #1044